documentation/index.md

```diff
@@ -7,12 +7,12 @@ In [Tawazi](https://pypi.org/project/tawazi/), there 3 Classes that will be mani
 `ExecNode` can take arguments and return values to be used as arguments in another object of type `ExecNode`.
 
 
-1. `DAG`: a wrapper around a function that defines a dag dependency.
-This function should only contain calls to objects of type `ExecNode`.<p></p>
+1. `DAG` / `AsyncDAG`: a wrapper around a function that defines a dag dependency.
+This function should only contain calls to objects of type `ExecNode` or to other `DAG`.<p></p>
```

Comment:
`or to other DAG` is not grammatically correct, wdym ?

Reply:
correct